Week-11/step-counter/src/App.js

```diff
@@ -0,0 +1,30 @@
+import './App.css';
+import AddStep from './Components/AddStep';
+import ResetSteps from './Components/ResetSteps';
+import {connect} from 'react-redux';
+import {addStep,resetStep} from './store/Actions/action';
+import { bindActionCreators } from 'redux'
+
+const mapStateToProps = state => {
+  return {
+    count : state.stepReducer.count
+  }
+}
+const mapDispatchToProps = dispatch => {
+  return bindActionCreators({
+    addStep,
+    resetStep
+  },dispatch)
+}
+
```

Comment:
These function should come in second last line before exporting. Again a suggestion.